openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/FeedCardHeader/FeedCardHeader.tsx

```diff
@@ -169,13 +183,61 @@ const FeedCardHeader: FC<FeedHeaderProp> = ({
       ].includes(entityType as EntityType)
     ) {
       displayName = getPartialNameFromFQN(entityFQN, ['service']);
+    } else if (
+      [EntityType.GLOSSARY, EntityType.GLOSSARY_TERM].includes(
+        entityType as EntityType
+      )
+    ) {
+      displayName = entityFQN.split(FQN_SEPARATOR_CHAR).pop();
```

Comment:
Do we need to join again by fqn separator character?

Reply:
No, here we will just need the glossary or glossaryTerm name.